playgrounds/dummy/app/components/DummyComponent.vue

```
@@ -0,0 +1,3 @@
+<template>
```

ESLint

[error] Component name "Dummy" should always be multi-word.

(vue/multi-word-component-names)